ethereum/erc/parsers.json

```diff
@@ -0,0 +1,145 @@
+{
+    "blockchainName": "ethereum",
+    "chainId": 1,
+    "contracts": [
+        {
+            "address": "0x7210000000000000000000000000000000000000",
+            "name": "ERC-721 Transfer",
+            "parsers": [
+                {
+                    "arguments": [
+                        {
+                            "format": "eip55",
+                            "name": "from",
+                            "value": {
+                                "address": {
+                                    "path": {
+                                        "default": "from"
+                                    }
+                                }
+                            }
+                        },
+                        {
+                            "format": "eip55",
+                            "name": "to",
+                            "value": {
+                                "address": {
+                                    "path": {
+                                        "default": "to"
+                                    }
+                                }
+                            }
+                        },
+                        {
+                            "format": "raw",
+                            "name": "tokenID",
+                            "value": {
+                                "uint256": {
+                                    "path": {
+                                        "default": "tokenId"
+                                    }
+                                }
+                            }
+                        }
+                    ],
+                    "functionName": "safeTransferFrom",
+                    "functionType": "transfer721",
```

Comment:
ERROR:validation.validators:Validation error: {'file': 'ethereum/erc/parsers.json', 'message': "'transfer721' is not one of ['stake', 'swap', 'wrap', 'unwrap', 'borrow', 'repay_loan', 'lend', 'withdraw', 'claim_reward', 'configure', 'buy', 'approve', 'read', 'transfer_erc721', 'transfer_erc1155']"}